composer.json

```diff
@@ -9,6 +9,9 @@
     "require": {
         "php": ">=7.1.0"
     },
+    "require-dev": {
+        "laravel/nova": "^4.0"
+    },
```

Comment:
Should this be added the the require section so that it will only work with Nova 4?